opm/autodiff/FlowMainEbos.hpp

```diff
@@ -318,13 +326,56 @@ namespace Opm
             streamLog->setMessageLimiter(std::make_shared<MessageLimiter>(10, limits));
             streamLog->setMessageFormatter(std::make_shared<SimpleMessageFormatter>(true));
 
-            // Read parameters.
             if ( output_cout_ )
-            {
-                OpmLog::debug("\n---------------    Reading parameters     ---------------\n");
+            { 
+            // Read Parameters.
+            OpmLog::debug("\n---------------    Reading parameters     ---------------\n");
             }
         }
-
+        
+        void printPRTHeader()
+        {
+            if ( output_cout_ )
+            { 
+            // Print header for PRT file.      
+            const std::string version = moduleVersionName();
+            const double megabyte = 1024 * 1024;
+            unsigned num_cpu = std::thread::hardware_concurrency();    
+            struct utsname arch;
+            char userName[64] = {0};
```

Comment:
This part is confusing: userName is never used. I think it would be better to do it like this (replacing the current lines 370-373):
```
const char* user = getlogin();
if (user) {
    ss << "User          =  " << user << std::endl;
}
```